saleor/graphql/schema.graphql

```diff
@@ -9939,6 +9943,13 @@ enum PageSortField {
   DEPRECATED: this field will be removed in Saleor 4.0.
   """
   PUBLISHED_AT
+
+  """
+  Sort pages by creation date.
+  
+  DEPRECATED: this field will be removed in Saleor 4.0.
```

Comment:
`npm run build-schema` command created wrong schema: it should say `Added in Saleor 3.8.` 
```
if self.name == "CREATION_DATE":
    description += DEPRECATED_IN_3X_INPUT
if self.name == "CREATED_AT":
    description += ADDED_IN_38
```
Same issue above for `PUBLISHED_AT`. It looks like, whenever a sorter is exactly the same, but different named, the problem occurs. Is it a known drawback of the `npm run build-schema` command?

Reply:
Going to be fixed in separate PR (https://github.com/saleor/saleor/issues/10968)